ghost/api-framework/lib/validators/input/all.js

```diff
@@ -126,7 +126,7 @@ module.exports = {
 
         // NOTE: this block should be removed completely once JSON Schema validations
         //       are introduced for all of the endpoints
-        if (!['posts', 'tags'].includes(apiConfig.docName)) {
+        if (!['posts', 'tags', 'mail_events'].includes(apiConfig.docName)) {
```

Comment:
Why didn't we go with the path that the comment above suggests - adding a "JSON Schema validation" for `mail_events`?

Reply:
Or rather "we should make it super obvious and easy to add validation schema" instead of having the hacky option as the only easy way to continue on with the work! 😂